griddedio/GriddedIO.F90

```diff
@@ -443,15 +461,22 @@ subroutine bundlepost(this,filename,oClients,rc)
 
         type(GriddedIOitemVectorIterator) :: iter
         type(GriddedIOitem), pointer :: item
+        logical :: have_time
+  
+        have_time = this%timeInfo%am_i_initialized()
 
-        this%times = this%timeInfo%compute_time_vector(this%metadata,rc=status)
-        _VERIFY(status)
-        ref = ArrayReference(this%times)
-        call oClients%stage_nondistributed_data(this%write_collection_id,trim(filename),'time',ref)
+        if (have_time) then
+           this%times = this%timeInfo%compute_time_vector(this%metadata,rc=status)
+           _VERIFY(status)
+           ref = ArrayReference(this%times)
+           call oClients%stage_nondistributed_data(this%write_collection_id,trim(filename),'time',ref)
 
-        tindex = size(this%times)
-        if (tindex==1) then
-           call this%stage2DLatLon(filename,oClients=oClients,_RC)
+           tindex = size(this%times)
+           if (tindex==1) then
+              call this%stage2DLatLon(filename,oClients=oClients,_RC)
+           end if
+        else
+           tindex = -1
```

Comment:
I see this tindex=-1 is an addition in bundlepost. I will give it try in swath to double check no effect on my code.